### DISCUSSION TITLE
Welcome to advanced-vfr-map Discussions!

### DISCUSSION THREAD

**Discussion by toorop** [flightsimcloud/advanced-vfr-map#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by llkj11** [comment]
Any plans on adding full integration with FS2020? Like displaying the flight path before flight to the VFR map? I don't know if its possible or not.

**Reply by toorop** [reply]
The main problem is that there is no doc, nor devtool, nor... from asobo/microsoft concerning UI.
We have to search, read available code, do reverse engineering, test, test ,test,...  it take a LOT of time.
To answer your question: we think it will be possible to display the fly path , but we don't fully know how... yet.
We are currently working on a new "engine" that will allow us better interaction between our code and the sim.
